### PR TITLE
[4.0] Extensions Discover

### PR DESCRIPTION
**Pull request by brianteeman**
Change the description of the extension from a tooltip to be visually displayed.

Remove the tooltip with the author url and email address as I don't see the value in displaying that information here

This improves accessibility

### After
image

### CODE REVIEW

**Comment by brianteeman** [comment]
In extensions manage #25226 I didnt display the information on screen like this. If requested this pr could be updated the same way and it would be like
image

File: administrator/components/com_installer/tmpl/discover/default.php [hunk 1]
```diff
@@ -100,9 +101,7 @@
 									<?php echo @$item->creationDate != '' ? $item->creationDate : '&#160;'; ?>
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/discover/default.php]:
```suggestion
									<?php echo !empty($item->creationDate) ? $item->creationDate : '&#160;'; ?>
```

File: administrator/components/com_installer/tmpl/discover/default.php [hunk 2]
```diff
@@ -100,9 +101,7 @@
...
 								</td>
 								<td class="d-none d-md-table-cell">
-									<span class="editlinktip hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', Text::_('COM_INSTALLER_AUTHOR_INFORMATION'), $item->author_info, 0); ?>">
 										<?php echo @$item->author != '' ? $item->author : '&#160;'; ?>
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/discover/default.php]:
```suggestion
									<?php echo !empty($item->author) ? $item->author : '&#160;'; ?>
```

**Comment by Quy** [comment]
Please apply `!empty` to line 98.

**Comment by brianteeman** [comment]
Thanks @quy all done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 2be0eb5e10321f870ac6a954c267ba458fa1450b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25224">issues.joomla.org/tracker/joomla-cms/25224</a>.</sub>

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on a9273797b68580602e96694997c9390a8ea82039<br /><br />Great improvement<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25224">issues.joomla.org/tracker/joomla-cms/25224</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25224">issues.joomla.org/tracker/joomla-cms/25224</a>.</sub>

**Comment by wilsonge** [comment]
Let's see how this goes. I'm happy with displaying on screen if we can - but I have a feeling for some extensions with longer descriptions we might find it necessary to move to tooltips to be honest

**Comment by wilsonge** [comment]
Thanks!

**Comment by tonypartridge** [comment]
That’s a good point. We should probably limit with a load more type toggle
👍

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**